freeipa/client/init.sls

```diff
@@ -6,6 +6,77 @@ include:
 - freeipa.client.nsupdate
 - freeipa.client.cert
 
+{%- if pillar.freeipa.client.install_principal is defined %}
```

Comment:
Refer `client` instead of pillar as it passes defaults substitution through map.jinja.